corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -359,6 +359,7 @@
     {% initial_page_data 'show_mobile_ux_warning' show_mobile_ux_warning %}
     {% initial_page_data 'toggles_dict' toggles_dict %}
     {% initial_page_data 'previews_dict' previews_dict %}
+    {% initial_page_data 'secure_cookies' secure_cookies %}
```

Comment:
Including this should mean you don't need to add it anywhere else, since virtually all HQ views ultimately derive from this template. But double-check the app preview view, that might be one of the few that doesn't.

Reply:
Ah right. I had this revelation on the Python side but not on the HTML side. Will make that change for anything that inherits from `base.html`. Thanks

Reply:
You might also consider adding this value somewhere in [context_processors](https://github.com/dimagi/commcare-hq/blob/master/corehq/util/context_processors.py) rather than adding it to individual views. It's definitely a tradeoff, since that'll add a little more bloat to all requests.